lib/values.js

```diff
@@ -0,0 +1,7 @@
+export function values(obj) {
+  let arr = [];
```

Comment:
Mowilem CI ze jest taka metoda `Object.keys` i w polaczeniu z `reduce` to byla by lepsza implementacja